src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { TwitterShareButton, TwitterIcon } from "react-share";
+import * as analytics from "./GoogleAnalytics";
+
+const title = "Helpful Flashcards for Developers @ ";
```

Comment:
Is this copy final? Can we think of something creative for sharing?

Reply:
Sure thing